### PR TITLE
fix: the failure of creating QuickSight dataset

### LINKED ISSUE #239: fail to create reporting for newly added application
## Summary

Failed to update reporting stack for creating dashboard for newly added application.

## Steps to reproduce

1. create a project with the pipeline
2. add a new application

Region: ap-northeast-1
Pipeline build number: v1.1.0-dev-main-202309181627-a15dafd3

## Relevant logs and/or screenshots

Got below log,

```
{
    "level": "INFO",
    "message": "start to create dataset",
    "sampling_rate": 1,
    "service": "ClickStreamAnalyticsOnAWS",
    "timestamp": "2023-09-19T07:50:15.825Z",
    "xray_trace_id": "1-650952b5-590c67e74a9302b21977081f"
}
{
    "level": "INFO",
    "message": "DatasetParameters: [{\"DateTimeDatasetParameter\":{\"ValueType\":\"SINGLE_VALUED\",\"TimeGranularity\":\"DAY\",\"Id\":\"a05a5ad5-c6b1-40de-9f96-f2bfcc5d2c21\",\"Name\":\"EventStartDate\"}}]",
    "sampling_rate": 1,
    "service": "ClickStreamAnalyticsOnAWS",
    "timestamp": "2023-09-19T07:50:15.825Z",
    "xray_trace_id": "1-650952b5-590c67e74a9302b21977081f"
}
{
    "level": "ERROR",
    "message": "Create QuickSight dataset failed due to: UnknownError",
    "sampling_rate": 1,
    "service": "ClickStreamAnalyticsOnAWS",
    "timestamp": "2023-09-19T07:50:17.002Z",
    "xray_trace_id": "1-650952b5-590c67e74a9302b21977081f"
}
```

## Possible fixes

---

This is :bug: Bug Report

### PR DESCRIPTION
**Pull request by llmin**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

fix #239 
fix QuickSight creation fail in ap-northeast-2

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [x] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: src/reporting/lambda/custom-resource/quicksight/index.ts [hunk 1]
```diff
@@ -406,12 +405,13 @@ const createDataSet = async (quickSight: QuickSight, awsAccountId: string, princ
       },
       LogicalTableMap: needLogicalMap ? logicalMap : undefined,
       ColumnGroups: colGroups.length > 0 ? colGroups : undefined,
-      DatasetParameters: props.datasetParameters,
```
Comment (zxkane) [inline src/reporting/lambda/custom-resource/quicksight/index.ts:409]: update test case to make sure we don't set this field

**Review by zxkane — CHANGES_REQUESTED** [review]
update end-to-end test result in PR description

File: src/reporting/lambda/custom-resource/quicksight/index.ts (hunk 1 above)
Comment (llmin) [reply src/reporting/lambda/custom-resource/quicksight/index.ts:409]: we can remove it from interface DataSetProps to make sure we don't use this parameter

File: src/reporting/quicksight-custom-resource.ts [hunk 2]
```diff
@@ -111,16 +111,6 @@ export function createQuicksightCustomResource(
             columnGeographicRoles: ['COUNTRY'],
           },
         ],
-        datasetParameters: [
-          {
-            DateTimeDatasetParameter: {
-              Id: 'a05a5ad5-c6b1-40de-9f96-f2bfcc5d2c21',
-              Name: 'EventStartDate',
```
Comment (zxkane) [inline src/reporting/quicksight-custom-resource.ts:118]: Did we move this criteria to SQL itself?

Comment (llmin) [reply src/reporting/quicksight-custom-resource.ts:118]: "datasetParameters" is a configuration used for adding parameters to the QuickSight dataset, other than SQL, is unnecessary and can be safely removed.

**Review by zxkane — APPROVED** [review]

**Pull request merged**